app/styleguide/page.tsx

```diff
@@ -1037,23 +1059,189 @@ function Page() {
+              variables also works. In addition to those, the following is also available:</p>
+            <dl>
+              <dt><code>--layout-gap</code></dt>
+              <dd>
+                This container uses flexbox, and this variable sets
+                the <code>gap</code> css property. Defaults to <code>1em</code>
+              </dd>
+            </dl>
+
+            <h2>Sidebar Container (Composite)</h2>
+            <p>This is a composite layout component that contains a sidebar
+              alongside a content container.</p>
+            <p><code>LayoutContainer &gt; LayoutWithSidebar</code></p>
+            <p><code>ContentContainer &gt; SidebarContainer</code></p>
+            <ToolbarContainer>
+              <Button onPress={toggleSidebar}>Toggle Sidebar</Button>
+            </ToolbarContainer>
+            <LayoutWithSidebar className="sg-sidebar">
+              <ContentContainer>
+                <h3>Layout With Sidebar</h3>
+                <p>This is the primary content for the slider component</p>
+
+                <div><pre><code>
+                  {`<LayoutWithSidebar>
+  <ContentContainer> ... </ContentContainer>
+  <SidebarContainer> ... </SidebarContainer>
+</LayoutWithSidebar>
+`}
+                </code></pre></div>
+              </ContentContainer>
+
+              <SidebarContainer isOpen={sidebarOpen}>
+                <p>This is the sidebar for the component</p>
+              </SidebarContainer>
+            </LayoutWithSidebar>
 
-            <h2>Layout</h2>
-            <p>TBD ...</p>
+            <p>
+              <code>LayoutWithSidebar</code> &nbsp;
+              can have a <code>ToolbarContainer</code> in addition to the sidebar.
+            </p>
+            <p>Note how in the previous example, the toolbar that demonstrates
+              the sidebar position and state, is <em>outside</em> &nbsp;
+              the <code>LayoutWithSidebar</code>, where in the example below, it's
+              part of it.</p>
+
+            <LayoutWithSidebar className="sg-sidebar">
+              <ToolbarContainer>
+                <p>This is a toolbar inside <code>LayoutWithSidebar</code></p>
+              </ToolbarContainer>
+
+              <ContentContainer>
+                <h3>Layout With Sidebar</h3>
+                <p>This is the primary content for the slider component</p>
+                <div><pre><code>
+                  {`<LayoutWithSidebar>
+  <ToolbarContainer> ... </ToolbarContainer>
+  <SidebarContainer> ... </SidebarContainer>
+  <ContentContainer> ... </ContentContainer>
+</LayoutWithSidebar>
+`}
+                </code></pre></div>
+              </ContentContainer>
+
+              <SidebarContainer>
+                <p>This is the sidebar for the slider component</p>
+              </SidebarContainer>
+            </LayoutWithSidebar>
+
+            <p>Example with a drawer instead of a sidebar</p>
+
+            <ToolbarContainer>
```

Comment:
When I tab around the sidebar drawer opens because of the "Close" button. Is that intentional? If not, you can use "excludeFromTabOrder" to prevent that from happening.

Reply:
I'm not getting this behaviour. Are you able to replicate it with the most recent update, and if so, write steps on how to reproduce?

Reply:
@andrewebdev, I was able to reproduce by highlighting the text "Example with a drawer instead of a sidebar", and then tabbing from there. Here is a recording. The drawer is opened just from me tabbing. I didn't click the "Toggle Drawer".

https://github.com/user-attachments/assets/63a9d064-675c-4930-80a9-76d838a6378e